3p/integration.js

```diff
@@ -266,7 +266,7 @@ import {zen} from '../ads/zen';
 import {zergnet} from '../ads/zergnet';
 import {zucks} from '../ads/zucks';
 import {speakol} from '../ads/speakol';
-
+import {mgid} from '../ads/mgid';
```

Comment:
Can we put this line (and the one above, my apologies), in alphabetical order? 😄